civis/ml/_model.py

```diff
@@ -192,6 +192,18 @@ def _exception_from_logs(exc, job_id, run_id, client, nlog=15):
     return exc
 
 
+def _get_credential(credential, client=None):
+    """Get the credential ID from either an integer or a string."""
```

Comment:
This will fail if the credential name is a string which can be cast to an integer. In addition, the documentation says that the input should be a name, rather than an ID. Could you remove this function and assume that the user inputs a credential name?